topology/probes/k8s/cronjob.go

```diff
@@ -25,85 +25,39 @@ package k8s
 import (
 	"fmt"
 
-	"github.com/skydive-project/skydive/logging"
-	"github.com/skydive-project/skydive/probe"
 	"github.com/skydive-project/skydive/topology/graph"
 
 	"k8s.io/api/batch/v1beta1"
-	"k8s.io/client-go/tools/cache"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 )
 
-type cronJobProbe struct {
-	defaultKubeCacheEventHandler
-	*kubeCache
-	graph *graph.Graph
+type cronJobHandler struct {
+	defaultResourceHandler
 }
 
-func dumpCronJob(cj *v1beta1.CronJob) string {
-	return fmt.Sprintf("cronjob{Namespace: %s, Name: %s}", cj.Namespace, cj.Name)
+func (h *cronJobHandler) GetRestClient(clientset *kubernetes.Clientset) rest.Interface {
```

Comment:
this will not work form istio which does not access a clientset (containing many rest-clients) but rather has two rest-clients (for istio networking and istio core) which is uses.